applications/openshift/api-server/api_server_admission_control_plugin_AlwaysAdmit/oval/shared.xml

```diff
@@ -17,10 +17,17 @@
     <ind:object object_ref="object_api_server_admission_control_plugin_AlwaysAdmit" />
   </ind:textfilecontent54_test>
 
+{{%- if product == "ocp4" %}}
+  <ind:textfilecontent54_object id="object_api_server_admission_control_plugin_AlwaysAdmit" version="1">
+    <ind:filepath>/etc/kubernetes/static-pod-resources/configmaps/config/config.yaml</ind:filepath>
+    <ind:pattern operation="pattern match">^.*"serviceAccountPublicKeyFiles":[\s]*\[.*"(\S+)".*\][,]*.*$</ind:pattern>
```

Comment:
```suggestion
    <ind:pattern operation="pattern match">^.*"serviceAccountPublicKeyFiles":[\s]*\[.*"(\S+)".*\]*.*$</ind:pattern>
```

Reply:
Should the regex evaluate for the final ``,``?

eg if this ends up as the last option in the stanza, the ``,`` will not be present.

Reply:
Not sure it makes sense for us to evaluate that atm. We can revisit later if it does.